### PR TITLE
PP-9925 add a bit more text to reason

### PR DESCRIPTION
**Pull request by SandorArpa**
- add a bit more explanation of what it used for to `reason`.

### CODE REVIEW

File: src/web/modules/transactions/update/upload.njk [hunk 1]
```diff
@@ -60,7 +60,7 @@
         </tr>
         <tr class="govuk-table__row">
           <th class="govuk-table__cell" scope="row"><span class="govuk-caption-m">reason</span></th>
-          <td class="govuk-table__cell">Only applies for disputes. If appears on payments or refunds, the upload shall be rejected</td>
+          <td class="govuk-table__cell">Only applies for disputes. Represents Stripe's reason of why the dispute has been opened. If appears on payments or refunds, the upload shall be rejected</td>
```
Comment (kbottla) [inline src/web/modules/transactions/update/upload.njk:63]: Any reason field `reason` is mandatory for disputes?

Comment (SandorArpa) [reply src/web/modules/transactions/update/upload.njk:63]: So reason is present incase we need to know, I guess

Comment (kbottla) [reply src/web/modules/transactions/update/upload.njk:63]: If `reason` is missing for a dispute, we can obviously add it using CSV but not required all the time as it is usually available on DISPUTE_CREATED event

Comment (SandorArpa) [reply src/web/modules/transactions/update/upload.njk]: Ok, I got it. Moved it to optional fields.

**Pull request merged**